code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -416,43 +384,17 @@
 	balding
 		name = "Balding Hair"
 		icon_state = "hair_e"
-		gender = MALE // turnoff!

```

Comment:
Is this on purpose?

Reply:
The to-do comment says to turn off the bit-flag, but no one got around to it.

Reply:
The to-do comment says that the hairstyle is a turnoff.